.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           key: ${{ runner.OS }}-${{ matrix.node }}-yarn-cache-${{ hashFiles('yarn.lock') }}
           restore-keys: ${{ runner.OS }}-${{ matrix.node }}-yarn-cache-
 
-      - run: yarn --no-progress --non-interactive --frozen-lockfile
+      - run: yarn install --immutable --immutable-cache --check-cache
```

Comment:
Link to the change – https://yarnpkg.com/cli/install#examples:

> Validate a project when using Zero-Installs (slightly safer if you accept external PRs) :
```bash
yarn install --immutable --immutable-cache --check-cache
```

Reply:
Removed cache flags because of this weird error – https://github.com/krzkaczor/ts-essentials/runs/4130609656?check_suite_focus=true